lib/ansible/modules/packaging/os/flatpak.py

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: flatpak
+version_added: "2.4"
+requirements:
+    - flatpak
+author:
+    - John Kwiatkoski (@jaykayy)
+short_description: Install and remove flatpaks.
+description:
+    - The flatpak module allows users to manage installation and removal of flatpaks.
+options:
+  name:
+    description:
+      - When I(state) is set to C(present), I(name) is best used as `http(s)` url format.
+        When set to C(absent) the same `http(s)` will try to remove it using the
+        name of the flatpakref. However, there is no naming standard between
+        names of flatpakrefs and what the reverse DNS name of the installed flatpak
+        will be. Given that, it is best to use the http(s) url for I(state=present)
+        and the reverse DNS I(state=absent). Alternatively reverse dns format can optimally
+        be used with I(state=absent), ex. I(name=org.gnome.gedit).
+    required: false
+    default: ''
+  remote:
+    description:
+      - The flatpak I(remote) repo to be used in the flatpak operation.
+    required: false
+    default: ''
```

Comment:
Why would you default to an empty string here ? Is an empty string actually useful ?
The default is None, which is a good default so you know it was set or not.
Simply remove this for the "default" default.